common/protob/messages-tron.proto

```
@@ -127,14 +144,20 @@ message TronSignedTx {
     optional bytes serialized_tx = 2; // Serialized transaction
 }
 
+enum TronMessageType {
+    V1 = 1 [deprecated=true];
```

Buf (1.60.0)

First enum value "V1" should have a numeric value of 0

(ENUM_FIRST_VALUE_ZERO)